src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerSourceConfig.java

```diff
@@ -67,6 +68,10 @@
   @Macro
   public String table;
 
+  @Description("The SELECT query to use to import data from the specified table")
+  @Macro
+  public String importQuery;
```

Comment:
If this is a required field it would break backwards compatibility. We should keep it optional and if user has not provided the import query we should be using existing behavior meaning - `select * from %s`

Reply:
updated PR